.github/workflows/macos.yml

```diff
@@ -205,24 +199,34 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
       with:
-        python-version: ${{ matrix.python-version }}
-        cache: 'pip'
+        python-version: "3.13"
 
-    - name: Build and Install SciPy
+    - name: Make gfortran-13 on runner image usable
       run: |
-        sudo xcode-select -s /Applications/Xcode_16.app
+        # Ensure we use gfortran-13 and that its runtime is on the library search path
+        FC=$(which gfortran-13)
+        GFORTRAN_LIB=$(dirname `$FC --print-file-name libgfortran.dylib`)
+        echo DYLD_LIBRARY_PATH=$GFORTRAN_LIB >> "$GITHUB_ENV"
+        echo FC=$FC >> "$GITHUB_ENV"
 
-        git submodule update --init
```

Comment:
Does the GH checkout update all submodules? Presumably this was originally added because it wasn't?